_sass/minimal-mistakes/_page.scss

```diff
@@ -2,560 +2,565 @@
    SINGLE PAGE/POST
    ========================================================================== */
 
-#main {
```

Comment:
Changes in this file are incomprehensible. Please revert.

Reply:
Something seems a little strange.
It was the 92nd line I was going to add the code below.

    h2:first-of-type {
      margin-top: 0em;
    }

The reason why I need to put that setting in is as follows.

[AS-IS]
The height of the profile panel on the left and the TOC panel on the right are the same, but the height of the content panel in the middle is slightly lower. 
This is because the margin-top of the h2 tag is applied.
The margin-top of the h2 tag should be applied from the use of the second h2 tag.

[TO-BE]
The margin-top of h2 used for the first time on the intermediate content panel shall not be set.

Reply:
But the `git diff` is still incomprehensible. Please revert the unnecessary part and make sure the diff accurately reflects your changes.

Reply:
Also, `:first-of-type` might not be as appropriate as `:first-child`, in case the content starts with something else. For example, I usually start my blog article with a regular paragraph.

Reply:
Then how about this?

  /* vertical alignment for first content */
    :first-child, 
    aside + :nth-child(2) {
      margin-top: 0em;
  }